andela/ah-the-jedi-backend
# #165305279 Get all bookmarked articles.

#### What does this PR do?
Fix the `GET` request for bookmarked articles
#### Description of Task to be completed?
- Fix the `return` statement to be outside the `for` loop.
#### How should this be manually tested?
1. Clone this repo.
   `$ git clone https://github.com/andela/ah-the-jedi-backend.git`
2. Fetch all origin to get all branches
   `$ git fetch origin`
3. Checkout to `ft-user-bookmark-article-165305279` branch.
   `$ git checkout ft-user-bookmark-article-165305279`
4. Activate `virtualenv`
    `$ virtualenv -p python .venv`
    `$ source .venv/bin/activate`
5. Install requirements
   `$ pip install -r requirements.txt`
6. Start server
   `$ python manage.py runserver`
7. Test the endpoint on postman in the following steps:
- Ensure the user is authenticated.
- Get the articles you bookmarked - `GET /api/articles/<slug>/bookmark/`
#### What are the relevant pivotal tracker stories?
[#165305279](https://www.pivotaltracker.com/story/show/#165305279)
#### Screenshot
Get all bookmarked articles
<img width="1097" alt="Screenshot 2019-05-08 at 16 56 52" src="https://user-images.githubusercontent.com/17095461/57380809-5b379f00-71b2-11e9-81e1-3b4dd5b55fd0.png">
